src/storage/src/hummock/utils.rs

```diff
@@ -96,7 +103,7 @@ where
     R: RangeBounds<B>,
     B: AsRef<[u8]>,
 {
-    ssts.filter(|info| filter_single_sst(info, key_range))
+    ssts.filter(|info| filter_single_sst(info, TableId::default(), key_range))
```

Comment:
We can pass the table_id in the method to do better pruning.

Reply:
> We can pass the table_id in the method to do better pruning.

fixed